controllers/flinkcluster_converter.go

```diff
@@ -14,6 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+// TODO FIND error causes
```

Comment:
@hjwalt what's this about?

Reply:
I missed removing that todo. It is the probe configuration tests. 

When updating the tests, I thought that the defaults will also be set by the converter if it is not currently set, turns out thats not the case. Whether or not we keep this behaviour is a separate discussion, so I didn't do a change on those points.

Reply:
Gotcha! Yeah defaults are only set through the webhook